graphium/finetuning/finetuning.py

```diff
@@ -0,0 +1,56 @@
+from typing import Iterable, List, Dict, Tuple, Union, Callable, Any, Optional, Type
+
+import pytorch_lightning as pl
+
+from torch.optim.optimizer import Optimizer
+from pytorch_lightning.callbacks import BaseFinetuning
+
+
+class MolecularFinetuning(BaseFinetuning):
+    def __init__(
+        self,
+        cfg,
+        train_bn: bool = False
```

Comment:
what is `train_bn`? Name seems ambiguous

Reply:
Ahhh I see! `train_bn` as in "should we continue training the bach_norm". Makes sense.